Wrappers/Python/cil/framework/utils.py

```diff
@@ -0,0 +1,13 @@
+def find_key(dic, val):
+    """return the key of dictionary dic given the value"""
+    return [k for k, v in dic.items() if v == val][0]
```

Comment:
I'm fairly sure this function can go.  It's not referenced anywhere within the codebase, except for within __init.py__.

Reply:
I agree